db/crud.go

```diff
@@ -913,15 +913,15 @@ func (db *DatabaseCollectionWithUser) updateHLV(d *Document, docUpdateEvent DocU
 
 // Updates or creates a document.
 // The new body's BodyRev property must match the current revision's, if any.
-func (db *DatabaseCollectionWithUser) Put(ctx context.Context, docid string, body Body) (newRevID string, doc *Document, err error) {
+func (db *DatabaseCollectionWithUser) Put(ctx context.Context, docid string, body Body) (newRevID string, cv Version, doc *Document, err error) {
```

Comment:
Given that this returns the updated *Document, is there a reason callers couldn't just retrieve cv from the HLV in the Document?